### PR TITLE
Upgrade to Debian 9 (stretch) and rearchitect build scripts

### PR DESCRIPTION
**Pull request by aursulis**

### CODE REVIEW

**Comment by aursulis** [comment]
@motiejus please review; @gustasthegreat can have a look through too for interest

File: config_overlay/archives/sublime.key.binary [hunk 1]
```diff
@@ -0,0 +1,62 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (motiejus) [inline config_overlay/archives/sublime.key.binary]: can we fetch this during build time?

File: config_overlay/archives/sublime.key.chroot [hunk 2]
```diff
@@ -0,0 +1,62 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (motiejus) [inline config_overlay/archives/sublime.key.chroot]: likewise, can we fetch this during build time?

File: config_overlay_contest/includes.chroot/usr/share/liox-config/d-i_hooks [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+set -e
+
+for P in $(ls /usr/share/liox-config/patches/*.patch); do
+    patch -d/ -p0 < ${P}
+done
+
+cp -aT /usr/share/liox-config/post-inst-overlay /
+
+cat >> /etc/hosts <<EOF
+34.243.103.175 olimp.cms.lmio.lt
```
Comment (motiejus) [inline config_overlay_contest/includes.chroot/usr/share/liox-config/d-i_hooks]: this should be a parameter to the image. Let's not leave IPs so deep.

**Pull request closed without merging**